bson/src/test/unit/org/bson/types/ObjectIdTest.java

```diff
@@ -104,7 +105,7 @@ public void shouldThrowIfMachineIdentifierIsTooLarge() {
     @Test
     public void testProcessIdentifier() {
         assertEquals(5, new ObjectId(0, 0, (short) 5, 0).getProcessIdentifier());
-        assertEquals(ObjectId.getGeneratedProcessIdentifier(), new ObjectId().getProcessIdentifier());
+        // assertEquals(ObjectId.getGeneratedProcessIdentifier(), new ObjectId().getProcessIdentifier());
```

Comment:
Since I removed the generation of the process ID, this assert is no longer valid.

Reply:
I think we'll be able to add this back

Reply:
I have added back the commented assert.